docs/_data/ruby.yml

```
@@ -1,3 +1,3 @@
-min_version: 2.5.0
-current_version: 3.1.3
-current_version_output: ruby 3.1.3p185 (2022-11-24 revision 1a6b16756e)
+min_version: 3.1.6
```

This change is not valid for the time being. Suggesting revert.

```suggestion
min_version: 2.5.0
```

---

Ah, ok. Got it. I'll revert and update the commit message since most of it no longer applies.